src/components/Form/FormSection.jsx

```diff
@@ -21,15 +19,7 @@ import { StatusNotification } from '../Notification';
  * {@link formSectionProps})
  */
 
-const FormSection = ({
-  title,
-  state,
-  showSpinner,
-  statusType,
-  defaultMessage,
-  file = null,
-  children
-}) => (
+const FormSection = ({ title, children }) => (
```

Comment:
Given the removal of the older notification props, think you could update the typedef of `formSectionProps` in `typedef.js`. See screenshot that they're still defining the old props.

<img width="1440" alt="Screen Shot 2023-08-28 at 14 41 41" src="https://github.com/codeforpdx/PASS/assets/14917816/19c36acd-07f6-4c8b-a18c-de75b4fcf413">

With the exception of title and children, think the rest of the definitions can be removed.

Reply:
Updated.